tests/pytests/runtests.sh

```diff
@@ -468,6 +416,7 @@ run_tests() {
 
 	local E=0
 	if [[ $NOP != 1 ]]; then
+	  cd $HERE
```

Comment:
Why is this needed?

Reply:
Probably due to the `cd` added in `setup_rejson`

Reply:
So after fixing the `setup_rejson` to exit at the same location we entered, can we remove it?

Reply:
yes, I removed it